### PR TITLE
CEP-1416: black as formatter

### PR DESCRIPTION
**Pull request by nemacysts**
this was mostly all done by the internal formatting script: all I did was re-format the .pre-commit-config.yaml file + tox.ini + add some `noqa`

`make test itest` worked just fine

### CODE REVIEW

**Review by vkhromov — CHANGES_REQUESTED** [review]
Please add `pyproject.toml` into the first commit.

**Comment by nemacysts** [comment]
> Please add `pyproject.toml` into the first commit.

ah, good catch: i went ahead and made the max line length match the previous config too: https://github.com/Yelp/clusterman/pull/114/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 120
+target_version = ['py36']
```
Comment (vkhromov) [inline pyproject.toml]:
```
target_version = ['py37']
```
?

**Review by vkhromov — APPROVED** [review]
lgtm overall, just 1 nit

File: pyproject.toml (hunk 1 above)
Comment (nemacysts) [reply pyproject.toml]: whoops!

**Pull request merged**